code_examples/sdk_examples/src/core_features/did/08_full_did_tx.ts

```diff
@@ -0,0 +1,19 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export async function signDidExtrinsic(
```

Comment:
Since it doesn't do much, it feels a little weird to wrap a `signDidExtrinsic`. The concern people have is how do you get these parameters.

Reply:
Most of these things should be separate guides tho.

* "the extrinsic that should be signed and submitted" if they don't have this, they don't need the guide. The extrinsic to submit will be something they find in other guides. and there we would link here. It doesn't make sense to explain where this comes from.
* SignCallback - We link to a guide on how to build this
* "the DID that wants to sign the extrinsic (also called origin of the extrinsic)" we should link to the DID creation guide.
* "and the address of the account that pays for the fees." here we should link to the concept guide that explains the difference between the DID and account and maybe how to create an account.

But explaining all this also in this guide would be too much